commands/core/site_install.drush.inc

```diff
@@ -55,7 +56,8 @@ function site_install_drush_command() {
         'example-value' => 'directory_name',
       ),
       'writable' => 'Make CMI and other dirs writable by both web and CLI users. Suitable for non Prod environments.',
-      'keep-config' => 'Keep CMI directories untouched. This preserves existing configuration.'
+      'keep-config' => 'Keep CMI directories untouched. This preserves existing configuration.',
+      'config' => 'After installation, override all configuration and UUIDs that were generated during installation with config from this file path.'
```

Comment:
"After a Minimal installation, import configuration from this path. Update site UUID accordingly."